kurtosis-module/impl/services/contract_helper_dynamodb.ts

```diff
@@ -0,0 +1,54 @@
+import { EnclaveContext, PortSpec, PortProtocol, ServiceID, ContainerConfig, ContainerConfigBuilder, ServiceContext } from "kurtosis-sdk";
+import log from "loglevel";
+import { Result, ok, err } from "neverthrow";
+import { getPrivateAndPublicUrlsForPortId, ServiceUrl } from "../service_url";
+
+const SERVICE_ID: ServiceID = "contract-helper-dynamo-db";
+const IMAGE: string = "amazon/dynamodb-local:1.20.0";
+
+// This is from the aws image and cannot be changes apparently
```

Comment:
I'd clarify that it's the _port number_ that can't be changed (port ID is our thing, so can always be changed)